### PR TITLE
adding signal whois for sender domain

### PR DESCRIPTION
**Pull request by bfilar**

### CODE REVIEW

File: signals/sender/sender_domain_whois_days_old.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: "Sender: Whois on sender domain"
+type: "query"
+source: |
+  beta.whois(sender.email.domain)
```
Comment (rw-access) [inline signals/sender/sender_domain_whois_days_old.yml:4]: I think you need to pick a property or two, right? Or can you handle signals like `{days_old: 3.14, found: true, registrar_name: "GoDaddy"}`

Comment (bfilar) [reply signals/sender/sender_domain_whois_days_old.yml:4]: To get the correct `days_old` value, I was going to grab `created_at`:
```
root_domain:sublimesecurity.com
created_at:2016-10-15T16:55:59Z
updated_at:2022-10-15T18:55:57Z
expires_at:2023-10-15T16:55:59Z
registrar_name:Google LLC
days_old:2529.7324520814623
found:true
```

Comment (bfilar) [reply signals/sender/sender_domain_whois_days_old.yml]: okay, after thinking it through I can deduce days_old for historic msgs in our training via alternative methods. Will go with `beta.whois(sender.email.domain).days_old` to ensure we return a single value

Comment (jkamdjou) [reply signals/sender/sender_domain_whois_days_old.yml]: we can always have more queries to return other fields too

Comment (rw-access) [reply signals/sender/sender_domain_whois_days_old.yml]: `days_old` is always relative to the MDM `_meta.created_at` field. So just make sure that field is right and you're golden.

Comment (bfilar) [reply signals/sender/sender_domain_whois_days_old.yml]: Awesome. Thanks

**Review by morriscode — APPROVED** [review]

**Pull request merged**